### DISCUSSION TITLE
http_requests_total, http_request_duration_seconds_sum, and http_request_duration_seconds_count

### DISCUSSION THREAD

**Discussion by collinkruger** [open-telemetry/opentelemetry-dotnet#4165] [Q&A] [answered]
Hi there,

I'm working on a polyglot system that uses OpenTelemetry/Prometheus, and there are a couple common metrics I see get generated automagically in other components.

- **http_requests_total**
- **http_request_duration_seconds_sum**
- **http_request_duration_seconds_count**

However, when I run the OpenTelemetry ASP.NET Core 6 Example project, all I see for ASP.NET/HttpClient are **http_client_duration_ms_bucket**, and **http_server_duration_ms_bucket**.

Do you know why this would be?
Ex: Do I need to turn on some environment variable to see the three metrics?

**Comment by cijothomas** [comment, marked as answer]
_http_client_duration_ms_bucket, and http_server_duration_ms_bucket._  

The above are expected from OpenTelemetry instrumentaion for HttpClient, Asp.NEt Core respective, and they follow OTel semantic conventions : https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md

OpenTelemetry does not have any convention that would result in "http_requests" metric, so it wont be part of OTel instrumented app. 
My best guess is that - its coming from a non-OTel component, like Prometheus Client in .NET/something else.

**Reply by collinkruger** [reply]
Thanks, Cijo!
